ddp-workspace/projects/ddp-atcp/src/app/providers/languages.provider.ts

```diff
@@ -0,0 +1,14 @@
+import { ValueProvider, InjectionToken } from '@angular/core';
+import languages from '../../assets/i18n/languages.json';
```

Comment:
Looks like TS unhappy with this line:
```
Cannot find module '../../assets/i18n/languages.json'. Consider using '--resolveJsonModule' to import module with '.json' extensionts(2732)
```
We should add `"resolveJsonModule": true` in the `compilerOptions` in the `ddp-workspace\tsconfig.json` https://hackernoon.com/import-json-into-typescript-8d465beded79
And rewrite import :
```
import * as languages from '../../assets/i18n/languages.json';

```